src/pages/ProductDetail/ProductDetail.jsx

```diff
@@ -1,10 +1,48 @@
 import React from 'react';
+import ProductCalculation from './component/ProductCalculation';
+import ProductImformation from './component/ProductImformation';
+import ProductSummary from './component/ProductSummary';
 import './ProductDetail.scss';
 
 const ProductDetail = () => {
   return (
     <div className="product-detail">
-      <div className="container">ProductDetail</div>
+      <div className="product-detail-container">
+        <img
+          className="product-detail-img"
+          src="./images/productDetail/sample.png"
+          alt="음식사진입니다."
+        />
+        <section className="product-detail-summary">
+          <div className="product-region">한국 / 아시아</div>
+          <h1 className="font-layout">땅콩소스로 숙성된 돼지갈비</h1>
+          <h1 className="font-layout">Pork ribs marinated in peanut sauce</h1>
```

Comment:
```suggestion
          <h1 className="font-layout kor-name">땅콩소스로 숙성된 돼지갈비</h1>
          <h1 className="font-layout eng-name">Pork ribs marinated in peanut sauce</h1>
```
className을 하나 더 추가해주시면 구분하기 더 좋을것같아요.

Reply:
css가 모두 동일한데 구분하기 위해 className만 추가해도 괜찮을까요??

Reply:
네 font-layout className으로 같은 속성이 다 적용되고 있고, kor-name, eng-name으로 속성을 부여하지않는한 
className만 추가한다고 css속성이 달라지진 않습니다